proto/common.proto

```diff
@@ -1151,9 +1151,11 @@ message CustomSpell {
 }
 
 message ItemSwap {
-	ItemSpec mh_item = 1;
-	ItemSpec oh_item = 2;
-	ItemSpec ranged_item = 3;
+	ItemSpec mh_item = 1 [deprecated=true];
+	ItemSpec oh_item = 2 [deprecated=true];
+	ItemSpec ranged_item = 3 [deprecated=true];
```

Comment:
We can use the `reserved` keyword to truly deprecate these with backwards-compatibility:
```
reserved 1, 2, 3;
resserved "mh_item", "oh_item", "ranged_item";
```